topics/dev/tutorials/conda/slides.html

```diff
@@ -818,7 +818,7 @@
   * fleeqtk can be built using `make` and installed with `make install`
 3. Use `conda build` to build the recipe
 4. Add a `requirement` for this new package in the example tool.
-5. Run `planemo conda_install --conda_use_local fleeqtk_seq.xml` to install the package for Galaxy
+5. Run `planemo conda_install --conda_use_local fleeqtk=1.3` to install the package for Galaxy
```

Comment:
This should not happen if you give the requirement a version as seen here: https://github.com/galaxyproject/planemo/blob/master/project_templates/conda_answers/exercise_2/fleeqtk_seq.xml#L3

Reply:
Hmmm something odd must of occurred then.  The version was specified in the xml.